lib/teracy-dev-core/config/provider.rb

```diff
@@ -0,0 +1,31 @@
+require 'teracy-dev/config/configurator'
+
+require_relative '../providers/manager'
+
+module TeracyDevCore
+  module Config
+    class Provider < TeracyDev::Config::Configurator
+
+      def configure_node(settings, config)
+        registry = Providers::Manager.registry
+        providers_settings = settings['providers'] ||= []
+        @logger.debug("providers_settings: #{providers_settings}")
+
+        providers_settings.each do |provider_settings|
+          next if !provider_settings['enabled']
+          providers = registry[provider_settings['type']]
+
+          if providers.nil?
+            @logger.warn("#{provider_settings['type']} have not registered")
```

Comment:
`@logger.warn("not any provider with the type of #{provider_settings['type']} is registered")`